sdk/unity/xr_unity_plugin/cardboard_xr_unity.cc

```diff
@@ -258,7 +258,8 @@ class CardboardApi::CardboardApiImpl {
     CardboardHeadTracker_resume(head_tracker_.get());
   }
 
-  void GetHeadTrackerPose(float* position, float* orientation) {
+  void GetHeadTrackerPose(CardboardViewportOrientation viewportOrientation,
```

Comment:
as said above, let's just set the value when calling `CardboardHeadTracker_getPose`. Please also left the TODO comment for supporting different orientations.